### ISSUE TITLE
Feature request: Tap on next direction should behave like destination

### ISSUE THREAD

**Issue by lightonflux** [osmandapp/Osmand#1650]
If you tap on the destination (red box) osmand shows you the position and flag on the map. Same for intermediate destinations.

But at the moment nothing will happen when you press on the direction arrows showed on the left (blue and green box. 

It would be very helpful for the navigator if pressing on it would show the turn so he/she can analyse the next turn before its too late. 

![212 60 217 13](https://cloud.githubusercontent.com/assets/1377943/9692972/159518f6-534d-11e5-8d7d-701474c1c538.png)

**Comment by sonora** [comment]
Victor, I agree, it would be a helpful feature in many situations!
Best, Hardy

**Comment by breversa** [comment]
I had no idea that tapping those boxes would display those pieces of information ! :-O
Is this behaviour documented somewhere ?

**Comment by vshcherb** [comment]
Yes if it wasn't taken by repeated announcement (click on next turn), which is useful as well for blind tapping. 
Let's think a bit about it.

**Comment by sonora** [comment]
Yes, I am aware of it. I think we could easily do both: Repeat the voice
prompt, and at the same time show the turn?

**Comment by vshcherb** [comment]
Usually it is bad idea, for consistency we should show location, but it is not that easy to change now.

**Comment by Barrocas** [comment]
I also agree on this sugestion, I believe it would be nice to preview the next turn or indication.
On my case I have set it up to center on my position after 5s, but we can always use the blue circle to return to our current location.
